app/.eslintrc.js

```diff
@@ -3,11 +3,12 @@ module.exports = {
   extends: '@react-native',
   plugins: ['eslint-plugin-no-inline-styles'],
   rules: {
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-shadow': 'off',
     'no-inline-styles/no-inline-styles': 'error',
     'react-native/no-inline-styles': 'off',
-    '@typescript-eslint/no-shadow': 'off',
     'react-native/no-unused-styles': 'error',
-    'react-native/no-raw-text': 'off',
+    'react-native/no-raw-text': 'off', //This rule is great, we don't enable it because of its performance. If we ever find similar rule we should enable it.
```

Comment:
```suggestion
    'react-native/no-raw-text': 'off', // This rule is great, we don't enable it because of its performance. If we ever find similar rule we should enable it.
```